entrypoint.sh

```diff
@@ -12,9 +12,9 @@ if ! [[ "$INPUT_POSTGRESQL_VERSION" =~ ^[0-9]+$ ]]; then
     exit 1
 fi
 
-# Check if the input is between 10 and 15 (inclusive)
-if (( INPUT_POSTGRESQL_VERSION < 10 || INPUT_POSTGRESQL_VERSION > 15 )); then
-    echo "Error: $INPUT_POSTGRESQL_VERSION is not between 10 and 15 (inclusive)."
+# Check if the input is between 10 and 16 (inclusive)
```

Comment:
Hi @BrennanColberg, I’ll suggest making version 11 the minimum version in that case.
```suggestion
# Check if the input is between 11 and 16 (inclusive)
```